src/main.rs

```diff
@@ -145,23 +145,29 @@ fn main() {
 
 /// Converts a JSON to an `avg::AVG`
 fn to_avg(data: &Value) -> avg::AVG {
+    let severity = data["severity"]
```

Comment:
Ok, this is the reason behind the issue I was describing before.

Here, we set the CVE severity to the severity of the AVG. Which isn't the same.

Reply:
As far as I can see we don't have that information from the json. So the best option is to return to the start commit 04aa1f5 or close the pull request.

Reply:
Or print the AVG and color only that. But this is outside the scope of this pull request.